dotCMS/src/main/java/com/dotcms/content/elasticsearch/util/RestHighLevelClientProvider.java

```diff
@@ -79,5 +79,7 @@ private static boolean disableStringIntern() {
     
     public abstract RestHighLevelClient getClient();
 
+    public abstract void rebuildClient();
```

Comment:
I would add a default impl for `rebuildClient()` that does nothing but log a message so customers who have overridden this class and have not seen issues will not have to rewrite their custom implementation to upgrade.

Reply:
Done